docs/05-infrastructure-nodes.md

```diff
@@ -0,0 +1,318 @@
+# OpenShift Infrastructure Nodes
+The OpenShift subscription model allows customers to run various core
+infrastructure components at no additional charge. In other words, a node
+that is only running core OpenShift infrastructure components is not counted
+in terms of the total number of subscriptions required to cover the
+environment.
+
+OpenShift components that fall into the infrastructure categorization
+include:
+
+* kubernetes and OpenShift control plane services ("masters")
+* router
+* container image registry
+* cluster metrics collection
+* cluster aggregated logging
+* service brokers
+
+Any node running a container/pod/component not described above is considered
+a worker and must be covered by a subscription.
+
+## More MachineSet Details
+In the cluster scaling exercises you explored the use of `MachineSets` and
+adding replicas within them. In the case of an infrastructure node, we want
+to create additional `Machines` that have specific kubernetes labels. Then,
+we can tell the various components to run specifically on hosts with those
+labels.
+
+To accomplish this, you will create additional `MachineSets`. The easiest way
+to do this is to `get` the existing `MachineSets` into a file, and then
+modify them. This is because the `MachineSet` has some details that are
+specific to the AWS region that the cluster is deployed in, like the AWS EC2
+AMI ID. For example, given the following output of `oc get machineset -n
```

Comment:
AMI IDs are probably going to become optional in `Machine(Set)s`... soon...  So maybe you want to motivate this by referencing the size/complexity of the YAML?  Or just use the AMI to motivate this approach now, and we'll revisit when AMIs become optional.

Reply:
> Yeah let's just leave it for now since that's the behavior at present?

That's fine with me, I just wanted to give you a heads-up ;).